log/wrapper.go

```diff
@@ -36,3 +35,28 @@ func TestWrapper(tb testing.TB) Wrapper {
 		tb.Errorf("logger called with msg: %q", msg)
 	}
 }
+
+// ZapWrapper wraps zap log package into a Wrapper.
+func ZapWrapper(logLevel zapcore.Level) Wrapper {
+	return func(msg string) {
+		switch logLevel {
+		default:
+			// for unknown values, fallback to info level.
+			fallthrough
+		case zapcore.InfoLevel:
+			Info(msg)
+		case zapcore.DebugLevel:
+			Debug(msg)
+		case zapcore.WarnLevel:
+			Warn(msg)
+		case zapcore.ErrorLevel:
+			Error(msg)
+		case zapcore.PanicLevel:
+			Panic(msg)
+		case zapcore.FatalLevel:
+			Fatal(msg)
+		case ZapNopLevel:
+			// do nothing
+		}
+	}
+}
```

Comment:
On second thought, for performance (it's not that performance in this function really matters that much, but ¯\\\_(ツ)\_/¯), let's do it this way instead:

```go
	// For unknown values, fallback to info level.
	f := Info
	switch logLevel {
	case zapcore.DebugLevel:
		f = Debug
	case zapcore.WarnLevel:
		f = Warn
	case zapcore.ErrorLevel:
		f = Error
	case zapcore.PanicLevel:
		f = Panic
	case zapcore.FatalLevel:
		f = Fatal
	case ZapNopLevel:
		return NopWrapper
	}
	return func(msg string) {
		f(msg)
	}
```
This way we only do the switch once, instead of every time the wrapper is called.

Reply:
looks good! thanks!